Makefile

```diff
@@ -232,7 +232,6 @@ docs-test-whitespace:
 .PHONY:docs-test-links
 docs-test-links: DOCS_FOLDERS := $(shell find . -name milv.config.yaml -exec dirname {} \;)
 docs-test-links:
-	go get -v github.com/magicmatatjahu/milv
```

Comment:
This should be left in to install `milv` for people who don’t have it.

Reply:
@webvictim The documentation building is Dockerized and milv is installed there. We're not installing mkdocs on every `make docs` do we?

Reply:
Good point. Our `Makefile` targets aren't explicitly always inside a Docker container, but I don't know of a situation where we're running this separately at the moment.